users/tracing/users.go

```diff
@@ -75,12 +75,20 @@ func (urm userRepositoryMiddleware) UpdatePassword(ctx context.Context, email, p
 	return urm.repo.UpdatePassword(ctx, email, password)
 }
 
-func (urm userRepositoryMiddleware) RetrieveAll(ctx context.Context, offset, limit uint64, ids []string, email string, um users.Metadata) (users.UserPage, error) {
+func (urm userRepositoryMiddleware) RetrieveAll(ctx context.Context, active bool, offset, limit uint64, ids []string, email string, um users.Metadata) (users.UserPage, error) {
```

Comment:
I'm afraid bool is not enough to represent all the states: `active`, `inactive`, `all`.

Reply:
Changed to using string

Reply:
wouldnt enum be more suitable